### DISCUSSION TITLE
Article

### DISCUSSION THREAD

**Discussion by anushkrishnav** [EddieHubCommunity/support#1100] [Show and tell]
https://anush-venkatakrishna.medium.com/part-1-winter-or-summer-take-your-baby-steps-into-opensource-now-7d661235d7ff

**Comment by kaiwalyakoparkar** [comment]
Hello Based on many git videos I have shrunk down the comman commands into a blog. Please check it out here: https://dev.to/kaiwalyakoparkar/useful-git-commands-you-need-to-know-lg2

**Comment by kaiwalyakoparkar** [comment]
Hello I have shifted to Dev.to as my primary blogging platform do checkout my profile : https://dev.to/kaiwalyakoparkar

**Comment by eddiejaoude** [comment]
@anushkrishnav I think your discussion would get better traction if the description was more informative than just a link
